linera-rpc/src/grpc/client.rs

```diff
@@ -256,6 +266,7 @@ impl ValidatorNode for GrpcClient {
 
         // The stream of `Notification`s that inserts increasing delays after retriable errors, and
         // terminates after unexpected or fatal errors.
+        let address = self.address.clone();
```

Comment:
Do you definitely need the clone here?

Reply:
I believe so because the closure below is `move`